web-app/src/components/ServicesPage/Service.js

```diff
@@ -8,7 +8,7 @@ import Address from '../Common/Address';
 class Service extends React.Component {
   constructor(props) {
     super(props);
-    this.id = props.id;
+    this.id = props.data.id;
```

Comment:
put ID on the service object so pass it with the whole service object as opposed to separately now